packages/terra-slide-panel/src/SlidePanel.module.scss

```diff
@@ -77,6 +77,17 @@
     }
   }
 
+  // Focus trap
```

Comment:
Since the user isn't necessarily trapped within the slide panel, maybe this could  say something like "Hide main"

Reply:
Replaced "Focus trap" with a description: "Remove main content from tab order when it's covered by slide panel on narrow screen"